app/views/past_events/index.html.erb

```diff
@@ -145,7 +126,6 @@
       <div class="time">
         <h4>2008</h4>
       </div>
-    </li>
 		<div style="clear: both;"></div>
-	</ul>
-</div>
\ No newline at end of file
+</div>
+<%= javascript_include_tag('fullpage/past_nuansa.js') %>
```

Comment:
I'm not sure why, but despite adding this same line in [app/views/layouts/application.html.erb](https://github.com/nuansa-cp/gapura/pull/23/files#diff-9599427925097c3c66f26ac1e0de5cad), the javascript won't load properly